README.md

```diff
@@ -107,6 +107,43 @@ In no specific order
 - Make it available on Maven
 - Add some validations based on feedback
 
-## Maven / Releases
-
-The tool has not been released yet. Your only option for now is to download the source code and build it.
+## Releases on maven central
+
+The different components of this project are released on Maven Central:
+
+The core library:
+
+```xml
+<dependency>
+  <groupId>org.openapitools.openapistylevalidator</groupId>
+  <artifactId>openapi-style-validator-lib</artifactId>
+  <version>${validatorVersion}</version>
+</dependency>
+```
+
+The command line tool (without dependency):
+
+```xml
+<dependency>
+  <groupId>org.openapitools.openapistylevalidator</groupId>
+  <artifactId>openapi-style-validator-cli</artifactId>
+  <version>${validatorVersion}</version>
+</dependency>
+```
+
+The command line tool is also available as standalone jar (containing its dependencies):
+
+```
```

Comment:
xml color highlighting is missing

Reply:
Fixed with https://github.com/OpenAPITools/openapi-style-validator/commit/e385cdec862518aaa73388d26a4243a821c0629f